proto/addressmetadata.proto

```diff
@@ -1,10 +1,34 @@
 syntax = "proto3";
 package models;
 
-// Basic key/value used to store header data.
-message Header {
-    string name = 1;
-    string value = 2;
+// Specifies the revocation method
+enum RevocationMethod {
+    /* If any of the pubkeys given in commitment_data[..n-1] signs
+    data commitment_data[n] then revocation is granted. */
+    BackupPubkey = 0;
+}
+
+// Message sent to server to revoke a key
+message Revocation {
```

Comment:
I'm not sure we need this specific type.  It can be included in the main addressmetadata type.  Makes it complicated to implement by having it multiple levels deep.